domains/dln.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Adityadln",
+        "email":"dlnaditya75@gmail.com"
+    },
+    "record": {
+        "CNAME": "Adityadln.github.io"
```

Comment:
"portfolio website coming soon....!!!"

you should complete your website first